pkg/hyperstart/hyperstart.go

```diff
@@ -73,6 +74,7 @@ var CodeList = map[string]uint32{
 	SetupInterface:  SetupInterfaceCode,
 	SetupRoute:      SetupRouteCode,
 	RemoveContainer: RemoveContainerCode,
+	PsContainer:     PsContainerCode,
```

Comment:
It is not defined, not yet I'm working on that https://github.com/clearcontainers/hyperstart/pull/30